app/schemas.py

```
@@ -867,4 +868,86 @@ class ImageOptimizationResponse(BaseModel):
+class ExchangeRateResponse(ExchangeRateBase):
+    id: str
+    created_at: datetime
+    updated_at: datetime
+    
+    class Config:
+        from_attributes = True
+        json_encoder = {
+            Decimal: lambda v: float(v) if v is not None else None,
+            datetime: lambda v: v.isoformat(),
+            date: lambda v: v.isoformat()
+        }        
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)